dist/leaflet.css

```diff
@@ -400,13 +404,14 @@ svg.leaflet-image-layer.leaflet-interactive path {
 
 .leaflet-container .leaflet-control-attribution {
 	background: #fff;
-	background: rgba(255, 255, 255, 0.7);
+	background: rgba(255, 255, 255, 0.8);

```

Comment:
@mourner This is potentially a fix for https://github.com/Leaflet/Leaflet/issues/7538, depending on the consensus of that issue. See https://github.com/Leaflet/Leaflet/issues/7538#issuecomment-1038285629.

Reply:
Yep! We can keep the issue to maybe consider bumping it further, but at least 0.8 seems like a definite improvement without compromising the design aesthetic much, so we can do it now.